src/utils.ts

```diff
@@ -172,16 +172,25 @@ const add = async (
 };
 
 export const run = async (
-  baseUrl: string,
-  username: string,
-  password: string,
+  baseUrl: string | undefined,
+  username: string | undefined,
+  password: string | undefined,
   pluginPath: string,
   pluginId: string | undefined,
   basicAuth: BasicAuth | null,
   proxy: string | undefined
 ): Promise<void> => {
   const browser = await launchBrowser(proxy);
   try {
+    if (!baseUrl) {
+      throw new Error("no base URL");
+    }
+    if (!username) {
+      throw new Error("no user name");
+    }
+    if (!password) {
+      throw new Error("no password");
+    }
```

Comment:
I think there is two things that needs to change.

**First, specify the reason why the error caused.**
`no ~~~` is not clear error message and partially overlapped yargs error.
Make it clear whether an error occurred because no arguments were specified, or an error occurred because the argument was undefined.

Example error message: `base-url is undefined!`

**Second, be same name as option name.**
`base url`  and `user name` is not appropriate. Keep in mind of writing consistent error message. 

Example is here.
- `base url` → `--base-url` or `base-url`
- `user name` → `--username` or `username`

Reply:
About the error message, I think we should unify it with that of yargs original message.